kubeflow/core/ambassador.libsonnet

```diff
@@ -170,6 +170,10 @@
                       },
                     },
                   },
+                  {
+                    name: "AMBASSADOR_SINGLE_NAMESPACE",
```

Comment:
What's this do? Does this mean Ambassador only handles services in this namespace?
If that's so then we should probably delete the service we created to create a forwarding rule to the K8s dashboard.